Our error checking doesn't work well with the parallel/sharded pytorch test.
As the overall gain is low, disable it and always run the full test suite in a single process.

Author: Alexander Grund (TU Dresden)

diff --git a/test/run_test.py b/test/run_test.py
index 9619cb2626e..ddfb200148f 100755
--- a/test/run_test.py
+++ b/test/run_test.py
@@ -815,7 +815,7 @@ def run_test_ops(test_module, test_directory, options):
     ]
     default_unittest_args.extend(rerun_options)
 
-    if 'slow-gradcheck' in os.getenv("BUILD_ENVIRONMENT", ""):
+    if True:
         extra_unittest_args = default_unittest_args.copy()
         # there are a lot of tests that take up a lot of space in slowgrad check, so don't bother parallelizing
         # it's also on periodic so we don't care about TTS as much
